test.py

```diff
@@ -153,6 +155,8 @@ def parse_cmd_line() -> argparse.Namespace:
                         help="timeout value for test.py/pytest session execution")
     parser.add_argument('--verbose', '-v', action='store_true', default=False,
                         help='Verbose reporting')
+    parser.add_argument('--quiet', '-q', action='store_true', default=False,
```

Comment:
just curios, if we already use pytest why aren't we using its args system directly ?

Reply:
Because they are not the same. It will pass all parameters, for example `-j` is for `test.py` but `pytest` have no such parameter. So it will fail to start. That's why we need to pass only parameters that are supported by `pytest`.